src/ProductConstructionService/ProductConstructionService.Api/Dockerfile

```diff
@@ -42,14 +44,16 @@ COPY ["src/Microsoft.DotNet.Darc/DarcLib", "./Microsoft.DotNet.Darc/DarcLib"]
 COPY ["src/ProductConstructionService", "./ProductConstructionService"]
 
 RUN dotnet build "./ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj" -c $BUILD_CONFIGURATION -o /app/build --no-restore
+RUN dotnet build "./ProductConstructionService/SubscriptionTriggerer/SubscriptionTriggerer.csproj" -c $BUILD_CONFIGURATION -o /app/build --no-restore
 
 FROM build AS publish
 ARG BUILD_CONFIGURATION=Release
 RUN dotnet publish "./ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj" -c $BUILD_CONFIGURATION -o /app/publish /p:UseAppHost=false
+RUN dotnet publish './ProductConstructionService/SubscriptionTriggerer/SubscriptionTriggerer.csproj' -c $BUILD_CONFIGURATION -o /app/publish /p:UseAppHost=false
```

Comment:
Do we want to publish this into the same folder as the above? Maybe not?

Reply:
I think it's fine, no real reason not to do it, we're gonna be calling the dlls manually anyway

Reply:
But isn't it overwriting some files maybe? If there would be some configs or similar?

Reply:
Ah right, we might be overwriting some runtime dll, I'll move it

Reply:
also updated all jobs with the new path

Reply:
@dkurepa I am not sure it is fixed? It still says `-o /app/publish`